client/api/omni/specs/infra.proto

```diff
@@ -29,6 +29,38 @@ message MachineRequestStatusSpec {
   string status = 4;
 }
 
+message InfraMachineSpec {
+  enum MachinePowerState {
+    POWER_STATE_OFF = 0;
```

Comment:
should it also be `unset` if the user doesn't want anything? (i.e. follow the defaults?)

Reply:
User's preference is here: https://github.com/siderolabs/omni/pull/707/files/d6063fcdb22fd49f859ba01bcae61a91b5f39343#diff-64ee15284d47f2291bfba2e406d10cb1aaf16ffc27e1af89008a114b58304c4eR1260-R1266

This is the final verdict, taking the user's preference and platform default (later to be defined in a resource). Not sure if it makes sense to add `unset` here.

Reply:
ah, ok, I was confused `preferred_power_state` name